db/migrate/20201112214758_create_contact_attempts.rb

```diff
@@ -0,0 +1,43 @@
+class CreateContactAttempts < ActiveRecord::Migration[6.0]
+  def up
+    create_table :contact_attempts do |t|
+      t.references :patient, index: true
+      t.references :user, index: true
+
+      t.boolean :successful, index: true
+      t.string :note
+
+      t.timestamps
+    end
+
+    # mysql does not support full joins
+    execute <<-SQL.squish
+      INSERT INTO contact_attempts (patient_id, user_id, successful, note, created_at, updated_at)
+      SELECT histories.patient_id as patient_id,
+            users.id as user_id,
+            histories.comment NOT LIKE '%unsuccessful%' as successful,
+            SUBSTRING_INDEX(histories.comment, 'uccessful contact attempt. Note: ', -1) as comment,
+            histories.created_at as created_at,
+            histories.updated_at as updated_at
+      FROM histories
+      LEFT JOIN users
+      ON histories.created_by = users.email
```

Comment:
This is more a comment for awareness: I think as of right now all the history `created_by` strings are just the users email, but that won't be the case after the ticket I'm working on. So this will work now, but I'm wondering if it's an issue that it wouldn't work in the future given that this migration. Pretty sure it's fine but just wanted to bring to it folks' attention.

Reply:
this will be fine because of the UNION thing